gamenews

```diff
@@ -1,4 +1,4 @@
-2022-12-03 13-00-00:
-	Title: Playtest 20221203
+2022-12-23 23-00-00:
+	Title: Playtest 20221223
 	Author: Lukas Franke
```

Comment:
You can change the name too if you want.

Reply:
If you insist